### PR TITLE
Create pytorch_siamese.py

### PR DESCRIPTION
**Pull request by Tuhin-Roy2311**
consistent accuracy of 0.908.. and same for F1, trained for 20 epochs

### CODE REVIEW

**Comment by Tuhin-Roy2311** [comment]
wrong one sorry
